DESCRIPTION

```diff
@@ -44,8 +50,8 @@ Description: Give advice about good practices when building R packages.
     Advice includes functions and syntax to avoid, package structure, code
     complexity, code formatting, etc.
 License: MIT + file LICENSE
-URL: https://mangothecat.github.io/goodpractice/, https://github.com/mangothecat/goodpractice
-BugReports: https://github.com/mangothecat/goodpractice/issues
+URL: https://ropensci-review-tools.github.io/goodpractice/, https://github.com/ropensci-review-tools/goodpractice
```

Comment:
why not docs.ropensci.org?

Reply:
https://github.com/ropensci/roregistry/blob/gh-pages/info/not_transferred.json

Reply:
https://github.com/ropensci/roregistry/pull/42 - do you want to merge that?

Reply:
Thanks for merging. Docs still won't be built until triggered by a change in main branch, and that won't happen until this PR is merged. My suggestion would be to first merge this, then subsequently update docs URL. Okay with you?

Reply:
Done.